develop/parachain-devs/parachain-launch/chain-spec.md

```diff
@@ -0,0 +1,113 @@
+
+The chain specification is defined using the [`ChainSpec` struct](https://paritytech.github.io/polkadot-sdk/master/sc_chain_spec/struct.GenericChainSpec.html){target=_blank}.
+
+!!!note "This page has a companion reference document"
+    To learn more about the particularities about chain specifications (and more specifically, the `chain-spec-builder` tool), [refer to the reference document.](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/chain_spec_genesis/index.html){target=_blank}
+
+The [`ChainSpec` struct](https://paritytech.github.io/polkadot-sdk/master/parachain_template_node/chain_spec/type.ChainSpec.html){target=_blank} separates the information required for a chain into two parts:
+
+- A client specification that contains information used by the _outer node_ to communicate with network participants and send data to telemetry endpoints. Many of these chain specification settings can be overridden by command-line options when starting a node or can be changed after the blockchain has started
+
+- The initial **genesis state** that all nodes in the network agree on. The genesis state must be established when the blockchain is first started and it cannot be changed thereafter without starting an entirely new blockchain
+
+## Customizing Outer Node Settings
+
+For the outer node, the chain specification controls information such as:
+
+- The boot nodes the node communicates with
+- The server endpoints for the node to send telemetry data to
+- The human and machine-readable names for the network the node connects to
+
+The chain specification can be customized to include additional information. For example, you can to configure the outer node to connect to specific blocks at specific heights to prevent long range attacks when syncing a new node from genesis.
+
+Note that you can customize outer node settings after genesis. However, nodes only add peers that use the same [`protocolId`](https://paritytech.github.io/polkadot-sdk/master/polkadot_parachain_lib/chain_spec/trait.ChainSpec.html#tymethod.protocol_id){target=_blank}.
+
+## Customizing the Genesis Configuration
+
+All nodes in the network must agree on the genesis state before they can agree on any subsequent blocks. The information configured in the genesis portion of a chain specification is used to create a genesis block. It takes effect when you start the first node and cannot be overridden with command-line options. However, you can configure some information in the genesis portion of a chain specification. For example, you can customize the genesis portion of the chain specification to include information such as:
+
+- Initial account balances
+- Accounts that are initially part of a governance council
+- The account that controls the `sudo` key
+- Any other genesis state for a pallet
```

Comment:
I would mention that this customization is feasible for development. For production raw format is recommended.